token/core/identity/storage.go

```diff
@@ -26,8 +38,15 @@ type Storage interface {
 	LoadMeta(identity view.Identity, meta any) error
 }
 
+type WalletPathStorage interface {
+	AddWallet(id string, path string) error
```

Comment:
We could simplify this signature to
```
Add(w WalletPath) error
GetAll() (Iterator[WalletPath], error)
```
Then we could even use generics, if this kind of storage is used in other places

Reply:
👍